deltasharing/src/test/scala/io/delta/sharing/spark/DeltaSharingUtilsSuite.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.sharing.spark
+
+import scala.reflect.ClassTag
+
+import org.apache.spark.{SharedSparkContext, SparkEnv, SparkFunSuite}
+import org.apache.spark.storage.BlockId
+
+class DeltaSharingUtilsSuite extends SparkFunSuite with SharedSparkContext {
```

Comment:
are you sure these tests are running??

Reply:
@vkorukanti can you validate that these test are running? 
my guess is that they are not, as the build does not add this module to the sparkGroup, and the sparkGroup determines which modules tests are being run under the spark test runner.

there is a choice to be made. 
1. either we add these sharing tests to the sparkGroup. same runner runs these tests but that runner will take longer (its already bad, many hours)
2. or we create a new group and a new test runner.

Reply:
Good catch. I only manually ran tests. 

let me know the preferred way.